slsDetectorSoftware/src/Detector.cpp

```diff
@@ -800,6 +800,25 @@ void Detector::startDetectorReadout() {
 
 void Detector::stopDetector(Positions pos) {
     pimpl->Parallel(&Module::stopAcquisition, pos);
+
+    // validate consistent frame numbers
+    switch (getDetectorType().squash()) {
```

Comment:
Do we need the switch? The others don't support setting frame number?

Reply:
Yes. Only Eiger, Jungfrau, ctb and moench have it implemented in the firmware. Ctb and moench were recent additions